qa/pull-tester/rpc-tests.py

```diff
@@ -30,6 +31,25 @@
 import tempfile
 import re
 
+# Parse arguments and pass through unrecognised args
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--coverage', action='store_true')
+parser.add_argument('-extended', action='store_true')
+parser.add_argument('--help', '-h', '-?', action='store_true')
+parser.add_argument('--parallel', type=int, default=4)
```

Comment:
If we are considering changing this around, my vote would be for renaming `--parallel` to be `--jobs` and also giving it a '-j' alias. The advantage is that it matches `make` where most people know what `-j3` means at a glance.